modules/acme-sh.nix

```diff
@@ -127,7 +132,14 @@ in
         mapDomain = name: dns: ''-d "${name}" --dns ${dns}'';
         primary = mapDomain mainDomain domains."${mainDomain}";
         domainsStr = lib.concatStringsSep " " ([primary] ++ (lib.remove primary (lib.mapAttrsToList mapDomain domains)));
-        cmd = ''acme.sh --server ${server} --issue ${lib.optionalString (!production) "--test"} ${domainsStr} --reloadcmd "touch ${statePath}/renewed" --syslog 6 > /dev/null'';
+        cmd = pkgs.writeShellScript "renew" ''
+          acme.sh --server ${server} --issue ${lib.optionalString (!production) "--test"} ${domainsStr} --syslog 6 > /dev/null
```

Comment:
as I see we have 2 commands. Does the writeShellScript apply by default `set -e`? Usually I add at the beginning to be sure?

Reply:
It doesn't add it automatically, good catch

Reply:
can we add the classical trio `set -euo pipefail`...even if there is no piping or variables